cgi-bin/DW/API/Key.pm

```diff
@@ -99,7 +99,8 @@ sub get_keys_for_user {
         push @keylist, $new;
     }
 
-    return \@keylist;
+    # sort oldest to newest (predictable ordering for management page)
+    return [ sort { $a->{keyid} <=> $b->{keyid} } @keylist ];
```

Comment:
nit: in principle, I don't love putting presentation logic down into the model layer, but this definitely doesn't hurt anything and does mean we don't have to repeat sorting logic in different areas... :handwaving: